app/models/display/route_line_direction.rb

```diff
@@ -23,16 +23,60 @@ def delay
 
     def max_actual_headway
       @max_actual_headway if @max_actual_headway
+      compute_actual_headways
```

Comment:
Let's revert changes of this file until the feature is implemented for routes as well.